api/commands.py

```diff
@@ -501,11 +503,40 @@ def add_qdrant_doc_id_index(field: str):
                     fg='green'))
 
 
+@click.command('create-tenant', help='Create account and tenant.')
+@click.option('--email', prompt=True, help='The email address of the tenant account.')
+@click.option('--language', prompt=True, help='Account language, default: en-US.')
+def create_tenant(email: str, language: Optional[str] = None):
+    """
+    Create tenant account
+    """
+    # Create account
+    email = email.strip()
```

Comment:
It seems that email is a required option. We'd better have a not-null check on its stripped string.

Reply:
fixed